cswinrt/strings/additions/Windows.Storage/IStorageFolderHandleAccess.cs

```diff
@@ -24,6 +24,8 @@ namespace ABI.Windows.Storage
 {
     using global::Microsoft.Win32.SafeHandles;
     using global::System;
+//TODO: Update netstandard version by adding objectreferenceattribute 
+#if NETSTANDARD2_0
```

Comment:
can we scope the #if blocks down to just the differences and not the entire class?  will make future maintenance easier